server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const app = express();
+const PORT = 3000;
```

Comment:
часто бывает полезно указывать порт в переменных окружения (ты столкнешься с этим, когда будешь упаковывать приложение в докер). Лучше написать так:
```suggestion
const PORT = process.env.PORT || 3000;
```

тогда, чтобы указать его в переменных окружения, есть 3 варианта 
1. Явно задать переменую (на маке ```export PORT=3000```)
2. 3апустить скрипт так ```PORT=3000 node server.js```
3. создать .env файл с содержимым ```PORT=3000```